NGitLab/Impl/ReleaseClient.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using NGitLab.Models;
+
+namespace NGitLab.Impl
+{
+    public class ReleaseClient : IReleaseClient
```

Comment:
Maybe add a small comment and a link to the documentation?

https://docs.gitlab.com/ee/api/releases/

Reply:
the others don't have it, feels odd to add it for this one.

Reply:
Well, you shouldn't feel bad to do better that other classes ;-)

Reply:
Fair enough 😸